types/prettier/index.d.ts

```diff
@@ -0,0 +1,38 @@
+// TODO: send PR?
+
+declare namespace Prettier {
```

Comment:
Could you briefly prep me on this one?

Reply:
When you installing some 3rd party dependency you usually want it’s type definitions too. From what I observed, there are two ways how to get them - library can ship `index.d.ts` file or community wrote it on their own and added to DefinitelyTyped repository. Since Prettier doesn’t have any type definition associated and I didn’t wanted to use `any` type I was looking for solutions how to declare custom types in a project for 3rd party lib. Although this seems to me so far as the best way, I’d explore this more.

Reply:
Excellent!